### PR TITLE
Categories

### PR DESCRIPTION
**Pull request by JesusEArambula**
Added some of the UI designing for the Home and About pages. Links from Home page don't route to anywhere. Also changed some of the styling for Resources page.

### CODE REVIEW

**Comment by francisli** [comment]
@JesusEArambula next time, after you see that a previous pull request has been merged, please remember to CREATE A NEW BRANCH from master, instead of continuing to work in the same branch. 

Branches should only exist for as long as they are needed to develop a new feature, have it reviewed, then merged. 

One of the big reasons to do this is because nowthis pull request contains BOTH the work you did previously (which has ALREADY been merged) AND the new work you did. So it makes it harder for me to just review the NEW work that you did since it's all mixed up together.

It also means that you missed on some changes on the master branch that are now causing the CONFLICTS that you see on your branch. By making sure you start a new branch on master, and keeping the lifetime of a branch limited, you reduce the changes for conflicting changes.

I will help resolve the conflicts on the branch. Again, this is adding more time/work on the reviewer's side since you didn't create a new branch from master. Then I will review the changes and provide feedback...

File: client/src/Home.js [hunk 1]
```diff
@@ -11,45 +15,47 @@ function Home() {
...
-            <h3>Step 1</h3>
-            <b>Learn About the Dream Act.</b>
-            <p>
-              The California Dream Act allows undocumented people who went to a california High School for 3 years or more, to get financial
-              aid from the state.
-            </p>
-            <a className="btn btn-primary" href="/">
-              Learn More
-            </a>
+        <Link onClick={() => setActive(false)} to="/resources">
```
Comment (francisli) [inline client/src/Home.js:18]: You've done a good job of styling the cards... but the goal is not to hard-code the categories, but to load them from the database and use code to generate the cards...!

**Review by francisli — CHANGES_REQUESTED** [review]
I've added comments to the PR for you to review...

The big change to make is to NOT hard-code the categories into the components, but use the API to load them from the database, and then use JavaScript/React to render the links in the header and the cards on the homepage DYNAMICALLY...

That way, if a category is created/updated/deleted from the database, the site will dynamically be updated, rather than having to edit the code and redeploy... that is the goal of why we did all this work to create the Category model...!

**Comment by francisli** [comment]
I see that in your hard-coded links, you've used urls for the categories like `/housing`, `/employment`, etc... if we want to be able to identify a category by a human-readable identifier like that, then we need to add it to the database- otherwise, right now the primary key ids in the database are numbers (i.e. 1, 2, 3, etc).

I'll add a migration to add a new column in the Categories table so that we can add this... sometimes called the "url slug" and then we can look up the category by that name in the database...

**Comment by francisli** [comment]
Ok, I've pushed up code that adds a unique url slug column to the Category model... make sure to log into the running server container and run `sequelize db:migrate` to make sure it gets added...

**Comment by francisli** [comment]
@haikelfd you merged your branch in but you didn't address my comments- why is there a dollar sign in front of the category names in the header?

**Comment by francisli** [comment]
@haikelfd I'm going to go ahead and remove that dollar sign from the header links...

**Review by francisli — APPROVED** [review]
Alright, let's merge and deploy where we are now to see how things are looking...!

**Comment by haikelfd** [comment]
@francisli, I'm facing a problem with my PC and I have limited access to the connection.  I tried to rewrite it with my MacBook. Which doesn't have the required specs to install a docker to test it out.  Hopefully, the heat problem will go away by next week and the blackouts will end. I will continue working on the website even after the intern ends.

**Comment by francisli** [comment]
@haikelfd ok, no worries, hope things get better with heat and power back home...!

**Pull request merged**